publishing_rate: 1
